### ISSUE TITLE
Plot vs. occurrence

### ISSUE THREAD

**Issue by tpoisot** [EcoJulia/SpatialEcology.jl#21]
Following up EcoJulia/GBIF.jl#6 -- I think a way to move from points to plot would be to create a "grid" of data. If we have a set of `GBIFRecords`, it can probably be done by looking at the lat/lon range and applying e.g. Sturge's rule to determine the number of bins (or letting the user specify breakpoints).

I'm agnostic as to whether this should be done here, or in GBIF.jl.

**Comment by mkborregaard** [comment]
Yes, you can grid occurrences already with `SpatialEcology.creategrid`(https://github.com/EcoJulia/SpatialEcology.jl/blob/master/src/Constructor_helperfunctions.jl#L135-L162). Interesting idea with the auto-binsize! I wonder if anyone would ever use that?

But the thing I think I understood was that actually many of GBIFs occurrence records are generated and uploaded as plot data, but that information is stripped as GBIF makes available as occurrences. But apparently they still do keep that information so it could be convenient to tap into it.

Congrats on getting the package out, it's really nice.
